mParticleCore.brs

```diff
@@ -1568,6 +1891,77 @@ function mParticleSGBridge(task as object) as object
+            logAdBreakStart:        function(mediaSession as object, adBreak as object, options={} as object) as object
+                                        return m.invokeFunction("media/logAdBreakStart", [mediaSession, adBreak, options])
+                                    end function,
+            logAdBreakEnd:          function(mediaSession as object, options={} as object) as object
+                                        return m.invokeFunction("media/logAdBreakEnd", [mediaSession, options])
+                                    end function,
+            logAdStart:             function(mediaSession as object, adContent as object, options={} as object) as object
+                                        return m.invokeFunction("media/logAdStart", [mediaSession, adContent, options])
+                                    end function,
+            logAdClick:             function(mediaSession as object, options={} as object) as object
+                                        return m.invokeFunction("media/logAdClick", [mediaSession, options])
+                                    end function,
+            logAdSkip:              function(mediaSession as object, options={} as object) as object
+                                        return m.invokeFunction("media/logAdSkip", [mediaSession, options])
+                                    end function,
+            logAdEnd:               function(mediaSession as object, options={} as object) as object
+                                        return m.invokeFunction("media/logAdEnd", [mediaSession, options])
+                                    end function,
+            logSegmentStart:        function(mediaSession as object, segment as object, options={} as object) as object
+                                        return m.invokeFunction("media/logSegmentStart", [mediaSession, segment, options])
+                                    end function,
+            logSegmentSkip:         function(mediaSession as object, options={} as object) as object
+                                        return m.invokeFunction("media/logSegmentSkip", [mediaSession, options])
+                                    end function,
+            logSegmentEnd:          function(mediaSession as object, options={} as object) as object
+                                        return m.invokeFunction("media/logSegmentEnd", [mediaSession, options])
+                                    end function,
+            logPlayheadPosition:    function(mediaSession as object, position as integer, options={} as object) as object
+                                        return m.invokeFunction("media/logPlayheadPosition", [mediaSession, position, options])
+                                    end function,
+            logQoS:                 function(mediaSession as object, startupTime as integer, droppedFrames as integer, bitRate as integer, fps as integer, options={} as object) as object
```

Comment:
make sure to test logQoS in the tests. there's 6 arguments here...want to make sure they all get passed through properly.  i think this will be fine, but there's got to be a better way of handling this:

![image](https://user-images.githubusercontent.com/5377436/87083852-49f58280-c1fb-11ea-8941-b44339058eba.png)